Source/Scene/Expression.js

```diff
@@ -62,7 +62,16 @@ define([
         asin : Math.asin,
         atan : Math.atan,
         radians : CesiumMath.toRadians,
-        degrees : CesiumMath.toDegrees
+        degrees : CesiumMath.toDegrees,
+        sign : Math.sign,
+        floor : Math.floor,
+        ceil : Math.ceil,
+        round : Math.round,
```

Comment:
It turns out that `round` isn't supported in GLSL ES 1.0.

In `Node.prototype.getShaderExpression` you will need to add a special case for it - you can substitute `round` with `floor(left + 0.5).`